Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -295,7 +295,7 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
                 code.Input = testCase.TestCaseInput;
                 var result = await ExecuteCodeAsync(code);
 
-                if(result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
+                if (result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
```

Comment:
That is a special error code returned by code base server when any server failure occurs. Currently it is not used as it should be but it gives us an indication that the docker or shell script has failed which can be used for error handling. 
BTW 34494 is numerical equivalent of 'ERROR' when you index each letter in QWERTY line of keys.